subprojects/base-services/src/main/java/org/gradle/internal/jvm/Jvm.java

```diff
@@ -359,7 +359,8 @@ private File findToolsJar(File javaHome) {
     }
 
     public boolean isIbmJvm() {
-        return false;
```

Comment:
IBM (or, rather, J9) runs on Apple as well, so the assumption that a JVM is either Apple, Ibm, or "Generic" doesn't hold anymore.